go_module/kotlin_exports/jni.c

```
@@ -0,0 +1,213 @@
+        // Если класс не найден (теоретически невозможно на Android)
+        return;
+    }
+
+    // 4. Ищем метод protect в найденном КЛАССЕ VpnService
+    // Сигнатура "(I)Z" — принимает int, возвращает boolean
+    g_protect_mid = (*env)->GetMethodID(env, vpn_cls, "protect", "(I)Z");
+
+    if (g_protect_mid == NULL) {
+        // Ошибка: метод не найден в классе VpnService
+        // Проверь, что твой объект в Kotlin действительно наследует VpnService
+    }
```

Cppcheck (2.20.0)

[error] Found an exit path from function with non-void return type that has missing return statement

(missingReturn)